e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -47,3 +59,52 @@ fixture.disablePageReloads`CheckBox`
     await createWidget('dxCheckBox', { value: undefined, text: 'indeterminate', rtlEnabled: true }, '#indeterminateRTL');
   });
 });
+
+['one two three', 'label'].forEach((text) => {
+  test(`Checkbox configurations label=${text}`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations label=${text}.png`, { element: '#container', shouldTestInCompact: true });
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations label=${text}.png`, { element: '#container', theme: getFullThemeName().replace('light', 'dark') });
+
+    await t
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  }).before(async () => {
+    await setStyleAttribute(Selector('#container'), 'padding: 5px; width: 1000px; height: 600px; column-count: 10;');
+
+    await insertStylesheetRulesToPage(`.${CHECKBOX_CLASS} { display: block; }`);
+
+    const stateClasses = [
+      READONLY_STATE_CLASS,
+      DEFAULT_STATE_CLASS,
+      ACTIVE_STATE_CLASS,
+      HOVER_STATE_CLASS,
+      FOCUSED_STATE_CLASS,
+      DISABLED_STATE_CLASS,
+    ];
+
+    for (const rtlEnabled of [false, true]) {
```

Comment:
a think this is too much for 1 test, it runs for 3-5 mins locally. I would split some checks, for example in 1 test we can check stateClasses with states, like cheked, unchecked etc, other would test the appearance with different width, the other one - different scaled container etc. and it would be easier to follow checkbox states to check them